client/packages/system/src/Program/Components/ProgramSearchModal/ProgramOptionRenderer.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  DefaultAutocompleteItemOption,
+  AutocompleteOptionRenderer,
+  Typography,
+  Box,
+} from '@openmsupply-client/common';
+import { ProgramDocumentFragment } from '../../api';
+
+export const getProgramOptionRenderer =
+  (): AutocompleteOptionRenderer<ProgramDocumentFragment> => (props, program) =>
+    (
+      <DefaultAutocompleteItemOption {...props}>
+        <Box display="flex" alignItems="flex-end" gap={1} height={25}>
+          <Box display="flex" flexDirection="row" gap={1} width={110}>
+            <Typography
+              overflow="hidden"
+              fontWeight="bold"
+              textOverflow="ellipsis"
+              sx={{
+                whiteSpace: 'no-wrap',
+              }}
+            >
+              {program?.name?.substring(0, 3)}
```

Comment:
there's nothing like a short code available, so I went with option (b) which is: hack something in and chat about it later.

Reply:
it looks nice with a code- and there is one in the mockup. but that could just be a copy from the suppliers / customers which has a code..